style.css

```diff
@@ -277,3 +277,1510 @@ li {
 .shape11:hover {
   background: none;
 }
+
+.container {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] Kindly ensure your main container uses `display: grid` to position elements on your page.